### ISSUE TITLE
"Show all components" button broken

### ISSUE THREAD

**Issue by Eternal-Rise** [vue-styleguidist/vue-styleguidist#1264]
**Current behavior**
"Show all components" button has broken link

**To reproduce**
1. Go to "Sections" example https://vue-styleguidist.github.io/sections/#/Components/Buttons?id=button
2. Open any component isolated (button "Open isolated")
3. Try to return back with button "Show all components"

**Expected behavior**
Expecting to return back to viewed component

**Comment by elevatebart** [comment]
Thank you @Eternal-Rise 

It seems the URL to return to the components is not the right one.

I'll try and fix it ASAP.

**Comment by Eternal-Rise** [comment]
Hi @elevatebart . I'm trying to fix that by myself, but can't run sections example locally. Do I miss something? 
![image](https://user-images.githubusercontent.com/34982358/154923579-f9793360-e43d-439d-98f9-8c48ba53fb24.png)


1. Clone the repo
2. Run `pnpm i`
3. Run `pnpm start sections`

**Comment by elevatebart** [comment]
You are absolutely right, I need to add a dependency to vue. this escaped my fixes when I migrated to pnpm.

Will fix now.

Thank you for taking the time.

[EDIT] example fixed and working

**Comment by Eternal-Rise** [comment]
Oh, still don't understand the development process. How may I apply my changes vue-styleguidist package for examples? 

1. Make changes
2. Run `pnpm compile`
2. Run `pnpm start`

But changes didn't applies for examples. Only compiles ts files
